src/core/server/saved_objects/export/get_sorted_objects_for_export.ts

```diff
@@ -121,6 +128,7 @@ async function fetchObjectsToExport({
       search,
       perPage: exportSizeLimit,
       namespaces: namespace ? [namespace] : undefined,
+      ...(workspaces ? { workspaces } : {}),
```

Comment:
does `{}` have same behaviour with `undefined`? will `{}` cause exported ndjson file contains workspace filed with empty array?

Reply:
You mean if workspaces is a `{}`, it will be exported in ndjson?

Reply:
I am not sure, most likely not, if we can have a test, that will be great.

Reply:
Oh, it is a input param and `{...({})}`is the same as workspaces: undefined but `{...({})}` is more clean as it won't have a workspaces: undefined in test snapshot.